apis/python/notebooks/tutorial_soma_append_mode.ipynb

```diff
@@ -673,6 +686,444 @@
         "    print()\n",
         "    print(X.used_shape())"
       ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "tags": []
+      },
+      "source": [
+        "## Ingesting multiple datasets to a SOMA Experiment"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "tags": []
+      },
+      "source": [
+        "Finally, we'll demonstrate combining multiple new AnnDatas into one experiment.\n",
+        "\n",
+        "The flow is pretty similar to the above:\n",
+        "\n",
```

Comment:
For the "Monday/Tuesday" model, the Tuesday is "new".

For the atlas model, let's stop calling the input AnnData objects "new". There is no reason for them to have to be "new". They can be quite "old". The point to be emphasized here is not a distinction between old and new. The point to be emphasized is that _multiple inputs_ can be ingested -- concatenated -- into a _single output_.